apps/ensadmin/src/components/recent-registrations/hooks.ts

```diff
@@ -1,17 +1,86 @@
-import { ensAdminVersion, selectedEnsNodeUrl } from "@/lib/env";
+import { ensAdminVersion } from "@/lib/env";
 import { useQuery } from "@tanstack/react-query";
-import { RecentRegistrationsResponse } from "./types";
+import { millisecondsInSecond } from "date-fns/constants";
+import { Address, getAddress, isAddressEqual } from "viem";
+import { Registration } from "./types";
 
 /**
- * Fetches info about the 5 most recently registered .eth domains that have been indexed.
- *
- * @param baseUrl ENSNode URL
- * @returns Info about the 5 most recently registered .eth domains that have been indexed.
+ * The data model returned by a GraphQL query for registrations.
+ */
+interface RegistrationResult {
+  registrationDate: UnixTimestampInSeconds;
+  expiryDate: UnixTimestampInSeconds;
+  domain: {
+    name: string;
+    createdAt: UnixTimestampInSeconds;
+    expiryDate: UnixTimestampInSeconds;
+    owner: {
+      id: Address;
+    };
+    wrappedOwner?: {
+      id: Address;
+    };
+  };
+}
+
+/**
+ * Numeric string representing a Unix timestamp in seconds.
```

Comment:
```suggestion
 * An integer value (representing a Unix timestamp in seconds) formatted as a string.
```

This states the idea more clearly. It's not so clear what a "Numeric string" is.